pitaya-sharp/NPitaya/src/Metrics/PitayaReporter.cs

```diff
@@ -36,51 +36,60 @@ static void RegisterCounterFn(IntPtr prometheusPtr, MetricsOpts opts)
             var ns = Marshal.PtrToStringAnsi(opts.MetricNamespace);
             // TODO (felipe.rodopoulos): prometheus-net does not support subsystem label yet. We'll use a hardcoded one.
             var name = Marshal.PtrToStringAnsi(opts.Name);
+            var help = Marshal.PtrToStringAnsi(opts.Help);
+            var labels = Marshal.PtrToStructure<string[]>(opts.VariableLabels);
```

Comment:
Did you test if this is working correctly? Printed the labels or something like this? My concern here is that C# strings are UTF-16 encoded and that's what `PtrToStringAnsi` does, read UTF-8 (Rust's default) and allocates and UTF-16 string but I don't know if this `PtrToStructure` will do the same, if not it will try to read the string as UTF-16.

Reply:
Solved. PTAL?